python-sdk/src/astro/databases/base.py

```diff
@@ -276,6 +276,61 @@ def drop_table(self, table: Table) -> None:
     # Table load methods
     # ---------------------------------------------------------
 
+    def create_schema_table_if_needed(
```

Comment:
Would renaming this to `create_schema_and_table_if_needed` make it more explicit?